pubsub-adapter-deployment.yml

```diff
@@ -0,0 +1,98 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: pubsub-adapter
+  labels:
+    app: pubsub-adapter
+    env: dev
+spec:
+  replicas: 1
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      app: pubsub-adapter
+  template:
+    metadata:
+      name: pubsub-adapter
+      labels:
+        app: pubsub-adapter
+        env: dev
+        version: "0.1.0dev"
```

Comment:
not `latest`?

Reply:
Yeah, we're not really making proper use of this label field anywhere, I just thought I'd make sure it was different for this "dev" version of the manifest.